supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -90,6 +204,22 @@ class TesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
     )
   }
 
+  override def scriptPreamble: String = {
```

Comment:
Can you add a comment here explaining the function of this preamble, what controls its use, etc?